ic10emu/src/grammar.rs

```diff
@@ -1029,6 +1070,7 @@ mod tests {
         define a_hash HASH(\"This is a String\")\n\
         alias a_var r0\n\
         alias a_device d0\n\
+        s d0 On 1\n\
```

Comment:
I'm honestly shocked that `On` isn't just one number, so I had to open up the game and play with it. Sure enough, you can't use `On` in random places that want numbers, and `LogicType.On` and `SlotLogicType.On` are two different numbers.

TIL

Reply:
Yeah it's honistly rather annoying to have to parse around but the tools are in place now